src/libertem/udf/base.py

```diff
@@ -62,6 +65,18 @@ def slice(self) -> Slice:
     def slice(self, new_slice: Slice):
         self._slice = new_slice
 
+    @property
+    def coordinates(self) -> List:
```

Comment:
Instead of `List` the coordinates could also be put into an `ndarray`; this would make "bulk actions" on them easier, and perform better for example in the case of whole-partition processing